### ISSUE TITLE
About the five-fold cross-validation

### ISSUE THREAD

**Issue by YizhaoX** [CodeGoat24/PLSNet#1]
Thanks to the authors for the excellent methodology. However, I noticed that the experimental validation method proposed in the original article is a five-fold cross-validation, while the data division method in the source code is just a random assignment. Please correct my misunderstanding if I am wrong.

source code:

    train_dataloader = utils.DataLoader(
        train_dataset, batch_size=dataset_config["batch_size"], shuffle=True, drop_last=False)

    val_dataloader = utils.DataLoader(
        val_dataset, batch_size=dataset_config["batch_size"], shuffle=True, drop_last=False)

    test_dataloader = utils.DataLoader(
        test_dataset, batch_size=dataset_config["batch_size"], shuffle=True, drop_last=False)

**Comment by sws827767542** [comment]
您好！不好意思，打扰了，感谢您代码的分享！我目前复现了您的这篇PLSNet论文，用的cc200图谱，在运行时，发现结果很不稳定，而且parser.add_argument('--repeat_time', default=100, type=int)这里的值应该设置为多少，100的话会不会跑的时间太长了。

**Comment by feile123** [comment]
> 您好！不好意思，打扰了，感谢您代码的分享！我目前复现了您的这篇PLSNet论文，用的cc200图谱，在运行时，发现结果很不稳定，而且parser.add_argument('--repeat_time', default=100, type=int)这里的值应该设置为多少，100的话会不会跑的时间太长了。

你好打扰一下，我想问一下，您有没有尝试加载一下结果的可视化这一步呢

**Comment by feile123** [comment]
> Thanks to the authors for the excellent methodology. However, I noticed that the experimental validation method proposed in the original article is a five-fold cross-validation, while the data division method in the source code is just a random assignment. Please correct my misunderstanding if I am wrong.
> 
> source code:
> 
> ```
> train_dataloader = utils.DataLoader(
>     train_dataset, batch_size=dataset_config["batch_size"], shuffle=True, drop_last=False)
> 
> val_dataloader = utils.DataLoader(
>     val_dataset, batch_size=dataset_config["batch_size"], shuffle=True, drop_last=False)
> 
> test_dataloader = utils.DataLoader(
>     test_dataset, batch_size=dataset_config["batch_size"], shuffle=True, drop_last=False)
> ```

你好打扰一下，我想问一下，您有没有尝试加载一下结果的可视化这一步了呢
